### PR TITLE
Make dropdown functional

### LINKED ISSUE #81: Make stateless components Functional
Replace all stateless components with functional components
**Additional context**
Are you working on this?
yes

### PR DESCRIPTION
**Pull request by ritwizsinha**
Made dropdown a functional component

Changed the size of dropdown

Fixes #81 

Screenshots for the change:
Screenshot from 2019-05-21 19-59-36

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@ritwizsinha Add a 10px padding in menu item

File: src/components/DropdownMenu/DropdownMenu.js [hunk 1]
```diff
@@ -1,21 +1,16 @@
...
 
-class DropdownMenu extends Component {
-  state = {
-    current: "mail"
-  };
-
-  render() {
-    const { username } = this.props;
-    const { isVendor, isAdmin } = this.props.user;
+function DropdownMenu(props) {
```
Comment (harsh-jindal) [inline src/components/DropdownMenu/DropdownMenu.js]: Use es6 function format `DropdownMenu = props => {}`

**Comment by harsh-jindal** [comment]
@ritwizsinha Just do the above mentioned small change and resolve conflicts.

File: src/components/DropdownMenu/DropdownMenu.css [hunk 2]
```diff
@@ -2,6 +2,19 @@
...
+.dropdown-margin{
+  margin: 20px 30px;
+  font-size: 30px;
+}
+.ant-dropdown-menu-item{
+  font-size: 20px;
+  font-family: monospace;
+  font-weight: 500;
+}
+.ant-drropdown-menu{
```
Comment (harsh-jindal) [inline src/components/DropdownMenu/DropdownMenu.css]: @ritwizsinha Add this padding to menu-item. Rest looks good 👍

Comment (ritwizsinha) [reply src/components/DropdownMenu/DropdownMenu.css]: done

**Review by harsh-jindal — APPROVED** [review]

**Pull request merged**